blockchain/lib/wallet.ts

```diff
@@ -1,7 +1,10 @@
 import { ethers } from "ethers";
 import { env } from "./env";
 import { getProvider } from "./provider";
+import config from "../hardhat.config"
 
-export function getWallet(): ethers.Wallet {
-  return new ethers.Wallet(env("ETH_PRIVATE_KEY"), getProvider());
+export function getWallet(network: string, accountNum = 0): ethers.Wallet {
```

Comment:
I could not get this to successfully work, without a runtime exception. I think this is due to `getProvider()` below being hardcoded to Ropsten, per the boilerplate.